qiskit/circuit/library/standard_gates/global_phase.py

```diff
@@ -0,0 +1,75 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Global Phase Gate"""
+
+from typing import Optional  # , Union
+import numpy
+
+# from qiskit.qasm import pi
```

Comment:
Please could you remove the commented-out code (here, and in a few other places in this file).